### ISSUE TITLE
does not work with ubuntu 18.04

### ISSUE THREAD

**Issue by vans163** [GetScatter/ScatterDesktop#285]
It just runs and hangs on the commandline.

**Comment by vans163** [comment]
seems when I installed it + ran it from file manager it worked. I did not want to install it so it pollutes my local directory with shortcuts, I wanted to run it from the .appimage directly.

Also it does not run from commandline when you execute it. I guess can close.

**Comment by nsjames** [comment]
Yeah I doubt that would work since it needs a local directory for persisting data. It might be possible to create a portable version of Scatter but we haven't really looked into it yet.
